engine-tests/src/utils/mocked_external.rs

```diff
@@ -76,27 +79,128 @@ impl near_vm_logic::External for MockedExternalWithTrie {
         self.underlying.storage_has_key(key, mode)
     }
 
-    fn generate_data_id(&mut self) -> near_primitives::hash::CryptoHash {
+    fn generate_data_id(&mut self) -> CryptoHash {
         self.underlying.generate_data_id()
     }
 
-    fn get_trie_nodes_count(&self) -> near_primitives::types::TrieNodesCount {
-        let db_reads = self.new_trie_node_count.get();
-        let mem_reads = self.cached_trie_node_count.get();
-        near_primitives::types::TrieNodesCount {
-            db_reads,
-            mem_reads,
-        }
+    fn get_trie_nodes_count(&self) -> TrieNodesCount {
+        self.underlying.get_trie_nodes_count()
```

Comment:
This is incorrect and is partially contributing to the lowered gas measurements. The default mock provided by nearcore [always returns 0 for touched trie nodes](https://github.com/near/nearcore/blob/master/runtime/near-vm-runner/src/logic/mocks/mock_external.rs#L142). In production Aurora has a large state so the number of trie nodes touched by state access is certainly not zero. This is the reason for the `new_trie_node_count` and `cached_trie_node_count` present in the original code.